fastlane/lib/fastlane/swift_fastlane_function.rb

```diff
@@ -271,12 +279,19 @@ def build_argument_list
         return "[]" # return empty list for argument
       end
 
-      argument_object_strings = @param_names.zip(param_type_overrides).map do |name, type_override|
+      argument_object_strings = @param_names.zip(param_type_overrides, param_default_values, param_optionality_values).map do |name, type_override, default_value, is_optional|
         sanitized_name = camel_case_lower(string: name)
         sanitized_name = sanitize_reserved_word(word: sanitized_name)
         type_string = type_override == :string_callback ? ", type: .stringClosure" : nil
 
-        "RubyCommand.Argument(name: \"#{name}\", value: #{sanitized_name}#{type_string})"
+        method_call = unless type_override == :string_callback || !is_optional 
+                        ".get()"
+                      else
+                        nil
+                      end 
```

Comment:
I think these double negatives are hard to follow 🙈 Perhaps inverting them will make it more easily readable, such as:

```suggestion
        method_call = if type_override != :string_callback || is_optional
                        ".get()"
                      else
                        nil
                      end 
```

note: I'm not rubyist so not sure if my logic is correct (e.g. idk if negating `==` in this case is the equivalent of `!=`)

Reply:
Yeah, got rubocop offenses for my double negation logic, but needed for fast testing the possible states. I'll try yours!